tendrl/commons/objects/cluster/atoms/check_cluster_available/__init__.py

```diff
@@ -0,0 +1,46 @@
+import etcd
+import gevent
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.commons.objects import AtomExecutionFailedError
+from tendrl.commons.utils import etcd_utils
+
+
+class CheckClusterAvailable(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(CheckClusterAvailable, self).__init__(*args, **kwargs)
+
+    def run(self):
+        retry_count = 0
```

Comment:
Use this logic https://github.com/Tendrl/commons/pull/646/files#diff-9668598ac287d05e33e98d925fdc78e6R251

instead of raw reads